Code/RDGeneral/Dict.h

```diff
@@ -94,6 +94,7 @@ class RDKIT_RDGENERAL_EXPORT Dict {
   Dict &operator=(const Dict &other) {
     _hasNonPodData = other._hasNonPodData;
     if (_hasNonPodData) {
+      reset();
```

Comment:
I'm considering renaming this to "clearNonPodData" for clarity.  "reset" isn't a good name.